frontend/src/index.tsx

```diff
@@ -26,6 +26,7 @@ const GlobalStyle = createGlobalStyle<{ theme: ThemeType }>`
   
   body {
     margin: 0;
+    height: 100%;
   }
```

Comment:
Could you add the styling for `<div id = "root">` here, instead of inline on the `index.html` file?